app/models/onebox_locale_site_setting.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# onebox_locale is just like any other locale setting, except it allows for an empty value,
+# which is used to indicate that the default_locale should be used for onebox_locale.
+class OneboxLocaleSiteSetting < LocaleSiteSetting
```

Comment:
Wonder if it'd be worth supporting a parameterized version of `enum` that can be configured with something like `allow_blank: true`. Not in scope for this change, but worth considering. 🙂

Reply:
Aye, that'd be a good general approach if we find that we need more `enum` settings that also allow an empty value.